modal/cli/volume.py

```diff
@@ -264,3 +264,18 @@ async def rm(
         if exc.status in (Status.NOT_FOUND, Status.INVALID_ARGUMENT):
             raise UsageError(exc.message)
         raise
+
+
+@volume_cli.command(name="cp", help="Copy a file within a volume.")
+@synchronizer.create_blocking
+async def cp(
+    volume_name: str,
+    paths: List[str],  # accepts multiple paths
```

Comment:
The comment should include the fact that the last path is treated as the destination. That's not knowable from just the type signature and the parameter name.